### ISSUE TITLE
Maxent suitabilities not plotting

### ISSUE THREAD

**Issue by danlwarren** [danlwarren/ENMTools#20]

**Comment by danlwarren** [comment]
Man, I just cannot figure out why this isn't plotting.  When you call plot() on an enmtools.maxent object manually it plots just fine, but it simply refuses to do it from the summary function.  Just weird.

**Comment by nahuron** [comment]
Is this still the case? I was able to get `summary(enmtools.maxent(my_model))` to produce the suitability plot with some of my data. If it is, I am happy to chase it down with a reproducible example!

**Comment by danlwarren** [comment]
Yeah it's weird because it works when you call summary, but not when you just call the object itself.  For instance:

test <- enmtools.maxent(iberolacerta.clade$species$monticola,
                        euro.worldclim)
test


test2 <- enmtools.gam(iberolacerta.clade$species$monticola,
                        euro.worldclim)
test2


The second one will show the suitabilities, the first one won't.  Both work when you explicitly call summary() or print(), but maxent models don't show anything when you just call the model itself.
